networkx/algorithms/clique.py

```diff
@@ -18,19 +18,21 @@
 #    All rights reserved.
 #    BSD license.
 from collections import deque
-from itertools import chain, islice
+from itertools import chain
+from itertools import combinations
+from itertools import islice
 try:
     from itertools import ifilter as filter
 except ImportError:
     pass
 import networkx
-from networkx.utils.decorators import *
+from ..utils import not_implemented_for
```

Comment:
The rationale for relative imports from [PEP 328][1]:

> Several use cases were presented, the most important of which is being able to rearrange the structure of large packages without having to edit sub-packages.

NetworkX is one of those "large packages".

[1]: https://www.python.org/dev/peps/pep-0328/#rationale-for-relative-imports